packages/commonwealth/client/styles/pages/login/login_mobile.scss

```diff
@@ -114,7 +114,7 @@
   background-size: cover;
   display: flex;
   flex-direction: column;
-  height: 100%;
+  height: 100vh;
```

Comment:
should we also set width to 100vw?

Reply:
Cool, I was thinking there might be a case where the height is fullscreen but the width doesn't cover full-screen width and shows the underlying screen on the left and right. But I couldn't reproduce it. Resolving